samples/Nancy.Demo.Hosting.AspnetCore/Program.cs

```diff
@@ -0,0 +1,15 @@
+﻿using Microsoft.AspNetCore;

```

Comment:
Can we move the using statements inside the namespace, to align with the rest of the Nancy code